Oracle.md

```diff
@@ -90,15 +90,13 @@ For example, if the rate at maturity time is $20000, the oracle will sign the di
 
 If the rate is greater than $999999, the oracle signs the digits `9`, `9`, `9`, `9`, `9`, `9`.
 
-### Serialization and signing of outcome values
+### Outcome signing
```

Comment:
Should probably be consistent and call these attestations to disambiguate from signing, might also be worth some `ctrl + f` to make sure this doesn't happen elsewhere

Reply:
Replaced everything with attestation and attest. Sounds a bit weird to me but let me know what you think.